Core/Core.Platform.Android/Core.Platform.Android.csproj

```diff
@@ -48,6 +48,14 @@
       <Project>{f1ea4bba-d775-45c0-99da-db0590bee87e}</Project>
       <Name>Core</Name>
     </ProjectReference>
+    <ProjectReference Include="..\..\Auth\Auth\Auth.csproj">
+      <Project>{55FF3B02-07D1-46F3-B0D5-584D3B98467E}</Project>
+      <Name>Auth</Name>
+    </ProjectReference>
+    <ProjectReference Include="..\..\Auth\Auth.Platform.Android\Auth.Platform.Android.csproj">
+      <Project>{ACEB8935-989F-46CA-8F0F-FBA3B358C70C}</Project>
+      <Name>Auth.Platform.Android</Name>
+    </ProjectReference>
```

Comment:
so the android core project now depends on the  platform specific implementation project of a service? 
If that is the case, I don't think we can do that, as we want to deliver the services as individual libraries and  developers can choose which lib they want to use. So it means the user may or may not use the `Auth` module. 

But with the setup like this, the `Auth` module is always required to even compile the core module. I don't think we should do that.